### PR TITLE
Translated toReversed() Method to Russian

### PR DESCRIPTION
**Pull request by AntonOshurek**
### Description

Translated the toReversed() method and its documentation into Russian. This commit adds the Russian translation for the method description, syntax, parameters, return value, examples, specifications, compatibility with browsers, and related sections. The translation ensures accessibility and comprehension for Russian-speaking developers accessing the toReversed() method documentation.

### Motivation

I aim to contribute to the open-source community by providing translations of documentation into Russian. By translating the toReversed() method documentation, I aim to make it more accessible and comprehensible for Russian-speaking developers. This contribution facilitates a more inclusive and diverse developer community by enabling users who prefer to read technical documentation in their native language to do so.

### CODE REVIEW

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Описание
+
+Метод `toReversed()` изменяет порядок элементов массива (первый элемент массива становится последним, а последний — первым.) и возвращает новый массив.
+
+Когда мы используем метод `toReversed()` на массиве с [пропущенными элементами](/ru/docs/Web/JavaScript/Guide/Indexed_collections#sparse_arrays), метод `toReversed()` проитерирует пустые элементы так, как если бы они имели значение `undefined`.
+
+## Примеры
+
+### обращение порядка элементов в массиве
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:34]:
```suggestion
### Обращение порядка элементов в массиве
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+### Использование toReversed() на массивах с пропущенными элементами
+
+Результатом вызова `toReversed()` никогда не будет массив с пропущенными элементами. Пустые ячейки в исходном массиве заменяются на `undefined` в возвращаемом массиве.
+
+```js
+console.log([1, , 3].toReversed()); // [3, undefined, 1]
+console.log([1, , 3, 4].toReversed()); // [4, 3, undefined, 1]
+```
+
+### вызов метода toReversed() на объектах которые не являются массивом
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:56]:
```suggestion
### Вызов метода toReversed() на объектах которые не являются массивом
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+{{Specifications}}
+
+## Совместимость с браузерами
+
+{{Compat}}
+
+## Смотрите также
+
+- [Polyfill of `Array.prototype.toReversed` in `core-js`](https://github.com/zloirock/core-js#change-array-by-copy)
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:81]:
```suggestion
- [Полифил `Array.prototype.toReversed` в `core-js`](https://github.com/zloirock/core-js#change-array-by-copy)
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+- [Indexed collections](/ru/docs/Web/JavaScript/Guide/Indexed_collections) guide
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:82]:
```suggestion
- Руководство по [упорядоченным наборам данных](/ru/docs/Web/JavaScript/Guide/Indexed_collections)
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+Нет.
+
+### Возвращаемое значение
+
+Новый массив, содержащий элементы в обратном порядке.
+
+## Описание
+
+Метод `toReversed()` изменяет порядок элементов массива (первый элемент массива становится последним, а последний — первым.) и возвращает новый массив.
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:28]:
```suggestion
Метод `toReversed()` изменяет порядок элементов массива (первый элемент массива становится последним, а последний — первым) и возвращает новый массив.
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
+---
+title: Array.prototype.toReversed()
+slug: Web/JavaScript/Reference/Global_Objects/Array/toReversed
+---
+
+{{JSRef}}
+
+## Сводка
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:8]:
```suggestion
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+Метод **`toReversed()`** является копирующей версией метода {{jsxref("Array/reverse", "reverse()")}}. Метод возвращает новый массив с элементами в обратном порядке.
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:10]:
```suggestion
Метод **`toReversed()`** возвращает новый массив с расположенными в обратном порядке элементами. Является **копирующей** версией метода {{jsxref("Array/reverse", "reverse()")}}.
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+Метод `toReversed()` изменяет порядок элементов массива (первый элемент массива становится последним, а последний — первым.) и возвращает новый массив.
+
+Когда мы используем метод `toReversed()` на массиве с [пропущенными элементами](/ru/docs/Web/JavaScript/Guide/Indexed_collections#sparse_arrays), метод `toReversed()` проитерирует пустые элементы так, как если бы они имели значение `undefined`.
+
+## Примеры
+
+### обращение порядка элементов в массиве
+
+В данном примере создается массив `items`, содержащий три элемента. Затем создается новый массив, в котором элементы расположены в обратном порядке относительно массива `items`. Массив `items` остается без изменений.
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:36]:
```suggestion
В данном примере создается массив `items`, содержащий три элемента. Затем создаётся новый массив, в котором элементы расположены в обратном порядке относительно массива `items`. Массив `items` остаётся без изменений.
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+console.log(items); // [1, 2, 3]
+
+const reversedItems = items.toReversed();
+console.log(reversedItems); // [3, 2, 1]
+console.log(items); // [1, 2, 3]
+```
+
+### Использование toReversed() на массивах с пропущенными элементами
+
+Результатом вызова `toReversed()` никогда не будет массив с пропущенными элементами. Пустые ячейки в исходном массиве заменяются на `undefined` в возвращаемом массиве.
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:49]:
```suggestion
В результате вызова `toReversed()` никогда не будет создан массив с пропущенными элементами. Пустые ячейки в исходном массиве заменяются на `undefined` в возвращаемом массиве.
```

File: files/ru/web/javascript/reference/global_objects/array/toreversed/index.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+```js
+console.log([1, , 3].toReversed()); // [3, undefined, 1]
+console.log([1, , 3, 4].toReversed()); // [4, 3, undefined, 1]
+```
+
+### вызов метода toReversed() на объектах которые не являются массивом
+
+В данном примере создается объект `arrayLike`, который не является обычным массивом, но обладает свойством `length` и целочисленными ключами. Поскольку `arrayLike` имеет свойство `length` равное 3, метод `toReversed()` создает новый массив длиной 3. Однако в `arrayLike` отсутствуют свойства с ключами '0' и '1'. Поэтому в возвращаемом массиве соответствующие элементы становятся `undefined`.
```
Comment (saionaro) [inline files/ru/web/javascript/reference/global_objects/array/toreversed/index.md:58]:
```suggestion
В данном примере создаётся объект `arrayLike`, который не является обычным массивом, но обладает свойством `length` и целочисленными ключами. Поскольку `arrayLike` имеет свойство `length` равное 3, метод `toReversed()` создает новый массив длиной 3. Однако в `arrayLike` отсутствуют свойства с ключами '0' и '1'. Поэтому в возвращаемом массиве соответствующие элементы становятся `undefined`.
```

**Review by saionaro — COMMENTED** [review]
@AntonOshurek спасибо за вклад в MDN!
Я внес несколько небольших правок

**Review by saionaro — APPROVED** [review]

**Comment by AntonOshurek** [comment]
Привет, внёс все необходимые изменения.

**Pull request merged**